ygen/protogen.go

```diff
@@ -1328,6 +1329,10 @@ func unionFieldToOneOf(fieldName string, e *yang.Entry, mtype *MappedType, annot
 	}
 	sort.Strings(typeNames)
 
+	if path == "" {
```

Comment:
Rather than add this here - suggest that we just make the caller where you explicitly specify `""` use args.field.Path()?

Reply:
Done.